### PR TITLE
HAWQ-1203. Ranger Plugin Service Implementation. (with contributions …

### PR DESCRIPTION
**Pull request by denalex**
…by Lav Jain and Leslie Chang)

### CODE REVIEW

File: ranger-plugin/integration/service/src/test/java/org/apache/hawq/ranger/integration/service/tests/RPSRequest.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
+package org.apache.hawq.ranger.integration.service.tests;
```
Comment (stanlyxiang) [inline ranger-plugin/integration/service/src/test/java/org/apache/hawq/ranger/integration/service/tests/RPSRequest.java]: license header

File: ranger-plugin/integration/service/src/test/java/org/apache/hawq/ranger/integration/service/tests/RPSResponse.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
+package org.apache.hawq.ranger.integration.service.tests;
```
Comment (stanlyxiang) [inline ranger-plugin/integration/service/src/test/java/org/apache/hawq/ranger/integration/service/tests/RPSResponse.java]: license header

File: ranger-plugin/integration/service/src/test/java/org/apache/hawq/ranger/integration/service/tests/Utils.java [hunk 3]
```diff
@@ -0,0 +1,57 @@
+package org.apache.hawq.ranger.integration.service.tests;
```
Comment (stanlyxiang) [inline ranger-plugin/integration/service/src/test/java/org/apache/hawq/ranger/integration/service/tests/Utils.java]: license header

**Comment by denalex** [comment]
updated the files that were missing licenses, created new PR: https://github.com/apache/incubator-hawq/pull/1092

**Pull request closed without merging**